### ISSUE TITLE
Not a valid add-on repository

### ISSUE THREAD

**Issue by spike3312** [pypolestar/polestar_api#157]
Hi,

I am new to Home assistant, this addon is pushing me into is as I can find no other way of getting the data out of the car without resorting to using 3rd party stuff, when i try to add the custom repository I get the following error.

https://github.com/leeyuentuen/polestar_api is not a valid add-on repository

**Comment by dgomes** [comment]
Out of curiosity, what 3rd party stuff have you considered ?

From the README:
```Add custom repository in HACS: https://github.com/leeyuentuen/polestar_api Search for integration 'polestar_api' ```

which means you first need to install HACS: https://hacs.xyz

**Comment by spike3312** [comment]
@dgomes Thanks for your reply, 

The other way I found was using Tibber but i'd rather not go through other places if I can get the data directly.

I use node red for most of my automation stuff but can not find a way of getting data into it easily.

I've installed the lastest HA and think I am following the instruction to add the repository correctly but always happy to be proven wrong.

![HA error](https://github.com/user-attachments/assets/460c73de-69cf-42a2-b101-e0408acd62a0)
![HA](https://github.com/user-attachments/assets/ead9f873-8df8-47e0-9786-1bedeba6c687)
